tests/conftest.py

```diff
@@ -74,10 +81,35 @@ def simple_dataset():
     return dataset
 
 
+@pytest.fixture(scope="session")
+def schema_name():
+    return pycds.get_schema_name()
+
+
+@pytest.fixture(scope="function")  # TODO: Can scope be broader?
+def database_uri():
+    """URI of test PG database"""
+    with testing.postgresql.Postgresql() as pg:
+        yield pg.url()
+
+
 @pytest.fixture
-def testconfig(testdb, request):
+def base_engine(database_uri):
+    """Plain vanilla database engine, with nothing added."""
+    yield create_engine(database_uri)
+
+
+@pytest.fixture
+def testdb(base_engine):
+    base_engine.execute("CREATE TABLE mytable (foo INTEGER, bar VARCHAR(50));")
```

Comment:
What is this for?

Reply:
This is an existing fixture for an existing [test](https://github.com/pacificclimate/pydap-extras/blob/4611e3a8f8c73ce57035267aad457d9448e43dac/tests/test_sql_handlers.py#L5-L26). In the Great Fixture Renovation, I folded it under the general tooling, but it hasn't actually changed. The upshot, I think, is that it sets up an ultra-minimal db for an ultra-minimal test of the SQL handler.